website/docs/ephemeral-resources/agent_token.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_agent_token"
+description: |-
+  Generates an ephemeral agent token.
+---
+
+# tfe_agent_token
+
+Generates a new agent token as an ephemeral value.
```

Comment:
I wonder if we should include a little here about the non-ephemeral-ness of the agent token? That we have chosen not to close/delete it, and the practical implications?